subsys/dfu/fmfu_fdev/Kconfig

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig FMFU_FDEV
+	bool "Full Modem Firmware Upgrade from flash device"
+	depends on CDDL_GEN
+
+
+if (FMFU_FDEV)
+
+choice
+	prompt "FMFU CDDL decoder build strategy"
+	default FMFU_CDDL_DECODER_BUILD_STRATEGY_SOURCE
+	help
+	  The metadata contained in the FMFU binary is encoded using CDDL.
+	  The decoder used for parsing this data is generated using cddl-gen.
+	  The generated decoder is committed to the repository since the format
+	  mostly will remain unchanged. The format is defined in the file
+	  'modem_update.cddl'. New source files must be generated if this file
+	  changes.
+
+config FMFU_CDDL_DECODER_BUILD_STRATEGY_SOURCE
+	bool "Use committed CDDL decoder source files"
+	help
+	  Select this option as long as 'modem_update.cddl' is unchanged.
+
+config FMFU_CDDL_DECODER_BUILD_STRATEGY_GENERATE
+	bool "Use cddl-gen python script to generate CDDL decoder source files"
```

Comment:
```suggestion
	bool "Generate CDDL decoder source files"
```

Reply:
"Re-generate"?

Reply:
I still prefer the existing use of `Generate` which were also used before my suggestion.